Source/WebCore/animation/WebAnimation.cpp

```diff
@@ -661,11 +661,17 @@ auto WebAnimation::playState() const -> PlayState
     return PlayState::Running;
 }
 
-Seconds WebAnimation::effectEndTime() const
+CSSNumberishTime WebAnimation::zeroTime() const
```

Comment:
Should this be a static method?

Reply:
Right now, it could be. But in a future patch (watch this space) we will query the attached timeline to determine whether we should return a percentage value or a time value.